src_c/color.c

```diff
@@ -1679,6 +1679,12 @@ _color_set_length(pgColorObject *color, PyObject *args)
 {
     int clength;
 
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "pygame.Color.set_length deprecated since 2.1.3",
```

Comment:
I feel like this is missing an "is" (before "deprecated")

Reply:
I think you don't need the 'is' here.